docs/topics/db/aggregation.txt

```diff
@@ -632,3 +637,34 @@ aggregate that author count, referencing the annotation field:
     >>> from django.db.models import Avg, Count
     >>> Book.objects.annotate(num_authors=Count("authors")).aggregate(Avg("num_authors"))
     {'num_authors__avg': 1.66}
+
+Handling empty querysets using aggregate ``default`` 
+----------------------------------------------------
+
+When an aggregate function is used with an empty ``QuerySet`` or grouping,
+None is returned as a default, this is because aggregate functions result 
+in NULL when used with an empty result set. The optional ``default`` argument 
+for aggregate functions allows specifying a value to use as a default when the 
+executed query returns no rows to perform an aggregation. Some aggregate functions 
```

Comment:
"...query returns no rows to perform an aggregation."

This ending doesn't make sense. Please rephrase this sentence.